en/messages.json

```diff
@@ -364,6 +364,23 @@
   "messageSplitTunnelBody": {
     "message": "Firefox is currently an “excluded app” in the Mozilla VPN application. To use this extension, open the application and go to Settings > Excluded Apps and remove Firefox.",
     "description": "Helptext body that is shown if the current Firefox is affected by Split-Tunneling"
+  },
+  "headerSubscriptionNotAvailable": {
+    "message": "Mozilla VPN Subscriptions are not yet available in your country",
```

Comment:
Why Subscriptions uppercase? In the next string is lowercase, which makes more sense since the rest is sentence case.

Reply:
Good call out! Changed that.